content/api/entity-reference/volunteer/index.html

```diff
@@ -134,32 +131,32 @@
             <tr>
                 <td>first_year_volunteered</td>
                 <td>integer</td>
-                <td>The first year volunteered.</td>
+                <td>This is a computed field based on the volunteer's first year volunteered.</td>
             </tr>
             <tr>
                 <td>total_hours</td>
                 <td>decimal</td>
-                <td>The total hours volunteered.</td>
+                <td>This is a computed field based on the volunteer's total hours volunteered.</td>
```

Comment:
I'd rework to avoid the passive "based on" formation. I'd change it to: "This computed field calculates the total hours that the constituent volunteered."